Build failure: pkgsi686Linux.vpl-gpu-rt

## Steps To Reproduce

Steps to reproduce the behavior:

```sh
nix build 'github:nixos/nixpkgs/23e89b7da85c3640bbc2173fe04f4bd114342367#pkgsi686Linux.vpl-gpu-rt'
```

Where `23e89b7d...` is the current `nixos-unstable` at the time of writing this.

## Build log

<!-- insert build log in code block in collapsable section -->

<details>

<summary>Build Log</summary>

(truncated to fit into github 65536 character limit, let me know if you need the full log)

```
[ 12%] Building ASM object contrib/ipp/CMakeFiles/ipp_sse4.dir/src/asm_intel64/picopyu8as.s.o
/build/source/contrib/ipp/src/asm_intel64/pcccopym7as.s:205: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/pcccopym7as.s:208: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/pcccopym7as.s:209: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/pcccopym7as.s:210: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/pcccopym7as.s:211: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/pcccopym7as.s:214: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/pcccopym7as.s:217: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/pcccopym7as.s:218: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/pcccopym7as.s:224: Error: bad register name `%r15'
/build/source/contrib/ipp/src/asm_intel64/pcccopym7as.s:227: Error: bad register name `%r14'
/build/source/contrib/ipp/src/asm_intel64/pcccopym7as.s:230: Error: bad register name `%r13'
/build/source/contrib/ipp/src/asm_intel64/pcccopym7as.s:233: Error: bad register name `%r12'
/build/source/contrib/ipp/src/asm_intel64/pcccopym7as.s:236: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/pcccopym7as.s:239: Error: bad register name `%rbp'
make[2]: *** [contrib/ipp/CMakeFiles/ipp_sse4.dir/build.make:792: contrib/ipp/CMakeFiles/ipp_sse4.dir/src/asm_intel64/pcccopym7as.s.o] Error 1
make[2]: *** Waiting for unfinished jobs....
cd /build/source/build/contrib/ipp && /nix/store/6plg0rh4cxk4vkafq3msycdv5923i4j1-gcc-wrapper-13.3.0/bin/gcc -DLINUX -DLINUX32 -DMEDIA_VERSION_STR=\"24.3.3\" -DMFX_API_VERSION=\"2.12+\" -DMFX_BUILD_INFO="\"Linux-6.6.53 6.6.53 | GNU 13.3.0 | glibc 2.40\"" -DMFX_GIT_COMMIT=\"\" -DMFX_ONEVPL -DMSDK_BUILD=\"\" -DNDEBUG -DONEVPL_EXPERIMENTAL -DSYNCHRONIZATION_BY_VA_SYNC_SURFACE -D_ARCH_IA32 -D_FILE_OFFSET_BITS=64 -D_P8 -D__USE_LARGEFILE64 -I/build/source/contrib/ipp/include -I/build/source/build -I/build/source/api/vpl -I/build/source/api/vpl/../mediasdk_structures -I/build/source/api/vpl/private -O3 -DNDEBUG -fPIC -Wno-deprecated-declarations -Wno-unknown-pragmas -Wno-unused -msse4.2 -o CMakeFiles/ipp_sse4.dir/src/asm_intel64/picopyu8as.s.o -c /build/source/contrib/ipp/src/asm_intel64/picopyu8as.s
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s: Assembler messages:
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:34: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:37: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:40: Error: bad register name `%r12'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:43: Error: bad register name `%r13'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:46: Error: bad register name `%r14'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:49: Error: bad register name `%r15'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:57: Error: bad register name `%rsi'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:60: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:61: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:62: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:63: Error: bad register name `%rsi'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:64: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:72: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:75: Error: bad register name `%rcx'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:80: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:81: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:82: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:83: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:84: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:91: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:92: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:93: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:94: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:96: Error: bad register name `%rsi'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:97: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:104: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:105: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:106: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:107: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:108: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:115: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:116: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:117: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:118: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:120: Error: bad register name `%rsi'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:121: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:128: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:132: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:133: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:134: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:139: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:140: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:142: Error: bad register name `%rsi'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:143: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:147: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:151: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:152: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:153: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:154: Error: bad register name `%rsi'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:155: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:162: Error: bad register name `%r15'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:165: Error: bad register name `%r14'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:168: Error: bad register name `%r13'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:171: Error: bad register name `%r12'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:174: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/piconvert02m7as.s:177: Error: bad register name `%rbp'
make[2]: *** [contrib/ipp/CMakeFiles/ipp_sse4.dir/build.make:818: contrib/ipp/CMakeFiles/ipp_sse4.dir/src/asm_intel64/piconvert02m7as.s.o] Error 1
In file included from /build/source/contrib/ipp/src/pvcdeinterlacevideoplane.c:32:
/build/source/contrib/ipp/src/pvccommonfunctions.h:37:50: error: unknown type name 'SIZE_T'
   37 |                                                  SIZE_T srcStep,
      |                                                  ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:39:50: error: unknown type name 'SIZE_T'
   39 |                                                  SIZE_T dstStep,
      |                                                  ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:47:51: error: unknown type name 'SIZE_T'
   47 |                                                   SIZE_T srcStep,
      |                                                   ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:49:51: error: unknown type name 'SIZE_T'
   49 |                                                   SIZE_T dstStep,
      |                                                   ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:57:50: error: unknown type name 'SIZE_T'
   57 |                                                  SIZE_T srcStep,
      |                                                  ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:59:50: error: unknown type name 'SIZE_T'
   59 |                                                  SIZE_T dstStep,
      |                                                  ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:67:38: error: unknown type name 'SIZE_T'
   67 |                                      SIZE_T srcStep,
      |                                      ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:69:38: error: unknown type name 'SIZE_T'
   69 |                                      SIZE_T dstStep,
      |                                      ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:82:51: error: unknown type name 'SIZE_T'
   82 |                                                   SIZE_T srcStep,
      |                                                   ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:84:51: error: unknown type name 'SIZE_T'
   84 |                                                   SIZE_T dstStep,
      |                                                   ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:92:52: error: unknown type name 'SIZE_T'
   92 |                                                    SIZE_T srcStep,
      |                                                    ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:94:52: error: unknown type name 'SIZE_T'
   94 |                                                    SIZE_T dstStep,
      |                                                    ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:102:51: error: unknown type name 'SIZE_T'
  102 |                                                   SIZE_T srcStep,
      |                                                   ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:104:51: error: unknown type name 'SIZE_T'
  104 |                                                   SIZE_T dstStep,
      |                                                   ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:112:39: error: unknown type name 'SIZE_T'
  112 |                                       SIZE_T srcStep,
      |                                       ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:114:39: error: unknown type name 'SIZE_T'
  114 |                                       SIZE_T dstStep,
      |                                       ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:127:51: error: unknown type name 'SIZE_T'
  127 |                                                   SIZE_T srcStep,
      |                                                   ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:129:51: error: unknown type name 'SIZE_T'
  129 |                                                   SIZE_T dstStep,
      |                                                   ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:137:52: error: unknown type name 'SIZE_T'
  137 |                                                    SIZE_T srcStep,
      |                                                    ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:139:52: error: unknown type name 'SIZE_T'
  139 |                                                    SIZE_T dstStep,
      |                                                    ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:147:51: error: unknown type name 'SIZE_T'
  147 |                                                   SIZE_T srcStep,
      |                                                   ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:149:51: error: unknown type name 'SIZE_T'
  149 |                                                   SIZE_T dstStep,
      |                                                   ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:157:39: error: unknown type name 'SIZE_T'
  157 |                                       SIZE_T srcStep,
      |                                       ^~~~~~
/build/source/contrib/ipp/src/pvccommonfunctions.h:159:39: error: unknown type name 'SIZE_T'
  159 |                                       SIZE_T dstStep,
      |                                       ^~~~~~
/build/source/contrib/ipp/src/pvcdeinterlacevideoplane.c: In function 'mfxiDeinterlaceFilterTriangle_8u_C1R':
/build/source/contrib/ipp/src/pvccommonfunctions.h:125:49: warning: implicit declaration of function 'deinterlace_filter_triangle_upper_slice_sse3'; did you mean 'deinterlace_filter_triangle_upper_slice'? [-Wimplicit-function-declaration]
  125 | #define deinterlace_filter_triangle_upper_slice deinterlace_filter_triangle_upper_slice_sse3
      |                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/contrib/ipp/src/pvcdeinterlacevideoplane.c:93:13: note: in expansion of macro 'deinterlace_filter_triangle_upper_slice'
   93 |             deinterlace_filter_triangle_upper_slice(pSrc,
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s: Assembler messages:
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:173: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:176: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:179: Error: `movslq' is only supported in 64-bit mode
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:180: Error: `movslq' is only supported in 64-bit mode
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:181: Error: `movslq' is only supported in 64-bit mode
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:182: Error: `movslq' is only supported in 64-bit mode
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:184: Error: bad register name `%r9'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:185: Error: bad register name `%r8'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:186: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:188: Error: bad register name `%rsi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:190: Error: bad register name `%rcx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:193: Error: bad register name `%r9'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:194: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:195: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:198: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:201: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:208: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:209: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s: /build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:210: Error: bad register name `%r8'
Assembler messages:
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:212: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:34: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:37: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:215: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:40: Error: bad register name `%r12'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:216: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:43: Error: bad register name `%r13'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:217: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:46: Error: bad register name `%r14'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:218: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:49: Error: bad register name `%r15'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:219: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:220: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:221: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:56: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:222: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:223: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:224: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:59: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:225: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:60: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:229: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:232: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:233: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:234: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:235: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:70: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:236: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:71: Error: bad register name `%rsi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:237: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:72: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:238: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:242: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:79: Error: bad register name `%rip)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:245: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:80: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:246: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:247: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:248: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:85: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:249: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:86: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:253: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:87: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:88: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:256: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:89: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:257: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:258: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:259: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:260: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:99: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:263: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:100: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:102: Error: bad register name `%rsi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:266: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:103: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:267: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:268: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:108: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:270: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:271: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:111: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:272: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:112: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:113: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:274: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:275: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:120: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:279: Error: bad register name `%r9'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:122: Error: bad register name `%rsi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:282: Error: bad register name `%rsi'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:123: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:283: Error: bad register name `%rcx'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:127: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:290: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:293: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:131: Error: bad register name `%rdi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:132: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:299: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:300: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:301: Error: bad register name `%r8'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:303: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:306: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:142: Error: bad register name `%rsi)'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:143: Error: bad register name `%rsi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:311: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:144: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:312: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:151: Error: bad register name `%r15'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:154: Error: bad register name `%r14'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:157: Error: bad register name `%r13'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:160: Error: bad register name `%r12'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:163: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:325: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/piconvert01m7as.s:166: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:329: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:330: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:331: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:332: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:333: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:334: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:335: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:336: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:337: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:338: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:339: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:342: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:346: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:349: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:350: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:351: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:352: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:353: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:354: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:355: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:359: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:362: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:363: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/pvccommonfunctions.h:135:50: warning: implicit declaration of function 'deinterlace_filter_triangle_middle_slice_sse3'; did you mean 'deinterlace_filter_triangle_middle_slice'? [-Wimplicit-function-declaration]
  135 | #define deinterlace_filter_triangle_middle_slice deinterlace_filter_triangle_middle_slice_sse3
      |                                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/contrib/ipp/src/pvcdeinterlacevideoplane.c:126:13: note: in expansion of macro 'deinterlace_filter_triangle_middle_slice'
  126 |             deinterlace_filter_triangle_middle_slice(pSrc,
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:364: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:365: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:366: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:370: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:373: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:374: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:375: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:376: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:377: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:381: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:384: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:385: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:386: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:387: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:388: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:392: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:395: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:396: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:397: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:398: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:399: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:403: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:406: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:407: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:411: Error: bad register name `%r9'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:414: Error: bad register name `%rsi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:415: Error: bad register name `%rcx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:422: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:425: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:431: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:434: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:435: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:436: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:439: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:442: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:443: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:444: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:447: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:450: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:451: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:452: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:455: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:458: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:459: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:460: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:463: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:466: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:467: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:468: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:471: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:474: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:475: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:476: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:477: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:478: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:484: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:485: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:486: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:487: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:488: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:489: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:490: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:491: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:492: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:499: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:500: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:501: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:502: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:503: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:504: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:505: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:506: Error: bad register name `%rbx'
make[2]: *** [contrib/ipp/CMakeFiles/ipp_sse4.dir/build.make:805: contrib/ipp/CMakeFiles/ipp_sse4.dir/src/asm_intel64/piconvert01m7as.s.o] Error 1
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:508: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:511: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:516: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:521: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:525: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:528: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:529: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:530: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:531: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:532: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:533: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:534: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:535: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:536: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:537: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:538: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:541: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:547: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:548: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:549: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:550: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:551: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:553: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:554: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:555: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:556: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:558: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:561: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:564: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:565: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:569: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:570: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:571: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:572: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:573: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:574: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:579: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:580: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:581: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:582: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:583: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:584: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:590: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:591: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:595: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:596: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:597: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:598: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:599: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:600: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:605: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:606: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:607: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:608: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:609: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:610: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:616: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:617: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:621: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:622: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:623: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:624: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:625: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:626: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:631: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:632: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:633: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:634: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:635: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:636: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:640: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:641: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:642: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:643: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:648: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:651: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:652: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:653: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:654: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:655: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:656: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:657: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:661: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:664: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:665: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:666: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:667: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:668: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:672: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:675: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:676: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:677: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:678: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:679: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:683: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:686: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:687: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:688: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:689: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:690: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:694: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:697: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:698: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:699: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:700: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:701: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:705: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:708: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:709: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:716: Error: bad register name `%rip)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:718: Error: bad register name `%rsp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:719: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:720: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:721: Error: bad register name `%rcx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:722: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:723: Error: bad register name `%r8'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:724: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:759: Error: bad register name `%rsp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:762: Error: bad register name `%r8)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:763: Error: bad register name `%r8'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:768: Error: bad register name `%r8)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:769: Error: bad register name `%r8'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:774: Error: bad register name `%r8)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:775: Error: bad register name `%r8'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:780: Error: bad register name `%r8)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:784: Error: bad register name `%rsp)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:789: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:792: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:795: Error: bad register name `%rsp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:802: Error: bad register name `%rbx)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:807: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:808: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:812: Error: bad register name `%rsp)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:815: Error: bad register name `%rsp)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:816: Error: bad register name `%rsp)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:817: Error: bad register name `%rsp)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:818: Error: bad register name `%rsp)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:819: Error: bad register name `%rsp)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:820: Error: bad register name `%rsp)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:821: Error: bad register name `%rsp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:824: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:827: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:829: Error: bad register name `%rbx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:834: Error: bad register name `%rdi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:835: Error: bad register name `%rdx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:836: Error: bad register name `%r8'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:839: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:842: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:843: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:844: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:845: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:846: Error: bad register name `%rax'
/build/source/contrib/ipp/src/pvccommonfunctions.h:145:49: warning: implicit declaration of function 'deinterlace_filter_triangle_lower_slice_sse3'; did you mean 'deinterlace_filter_triangle_lower_slice'? [-Wimplicit-function-declaration]
  145 | #define deinterlace_filter_triangle_lower_slice deinterlace_filter_triangle_lower_slice_sse3
      |                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/contrib/ipp/src/pvcdeinterlacevideoplane.c:167:13: note: in expansion of macro 'deinterlace_filter_triangle_lower_slice'
  167 |             deinterlace_filter_triangle_lower_slice(pSrc,
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:852: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:855: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:858: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:859: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:860: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:863: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:864: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:868: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:869: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:870: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:871: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:872: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:878: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:882: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:885: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:887: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:891: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:895: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:896: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:897: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:898: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:899: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:903: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:907: Error: bad register name `%r10)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:908: Error: bad register name `%r10'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:909: Error: bad register name `%r11)'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:910: Error: bad register name `%r11'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:911: Error: bad register name `%rax'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:916: Error: bad register name `%r9'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:919: Error: bad register name `%rsi'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:920: Error: bad register name `%rcx'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:928: Error: bad register name `%rbp'
/build/source/contrib/ipp/src/asm_intel64/picopyu8as.s:931: Error: bad register name `%rbx'
make[2]: *** [contrib/ipp/CMakeFiles/ipp_sse4.dir/build.make:831: contrib/ipp/CMakeFiles/ipp_sse4.dir/src/asm_intel64/picopyu8as.s.o] Error 1
/build/source/contrib/ipp/src/pvccommonfunctions.h:155:37: warning: implicit declaration of function 'deinterlace_filter_triangle_sse3'; did you mean 'deinterlace_filter_triangle'? [-Wimplicit-function-declaration]
  155 | #define deinterlace_filter_triangle deinterlace_filter_triangle_sse3
      |                                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/contrib/ipp/src/pvcdeinterlacevideoplane.c:215:13: note: in expansion of macro 'deinterlace_filter_triangle'
  215 |             deinterlace_filter_triangle(pSrc,
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [contrib/ipp/CMakeFiles/ipp_sse4.dir/build.make:765: contrib/ipp/CMakeFiles/ipp_sse4.dir/src/pvcdeinterlacevideoplane.c.o] Error 1
make[2]: Leaving directory '/build/source/build'
make[1]: *** [CMakeFiles/Makefile2:470: contrib/ipp/CMakeFiles/ipp_sse4.dir/all] Error 2
make[1]: Leaving directory '/build/source/build'
make: *** [Makefile:139: all] Error 2
```

</details>

## Notify maintainers

@evanrichter
@PJungkamp

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/vp/vpl-gpu-rt/package.nix
+++ b/pkgs/by-name/vp/vpl-gpu-rt/package.nix
@@ -29,10 +29,7 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/intel/vpl-gpu-rt";
     changelog = "https://github.com/intel/vpl-gpu-rt/releases/tag/${src.rev}";
     license = [ lib.licenses.mit ];
-    platforms = lib.platforms.linux;
-    # CMake adds x86 specific compiler flags in <source>/builder/FindGlobals.cmake
-    # NOTE: https://github.com/oneapi-src/oneVPL-intel-gpu/issues/303
-    broken = !stdenv.hostPlatform.isx86;
+    platforms = [ "x86_64-linux" ];
     maintainers = with lib.maintainers; [ evanrichter pjungkamp ];
   };
 }

